x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -47,6 +47,11 @@ export const LATEST_FINDINGS_INDEX_DEFAULT_NS =
   'logs-cloud_security_posture.findings_latest-default';
 export const LATEST_FINDINGS_RETENTION_POLICY = '26h';
 
+export const KSPM_RETENTION_POLICY = '5h';
```

Comment:
following up on the @opauloh 's [comment](https://github.com/elastic/kibana/pull/160313#discussion_r1390084814) I also don't really understand why exactly these retention policies if we have the following retention on the `latest` transforms

```
export const LATEST_FINDINGS_RETENTION_POLICY = '26h';
export const LATEST_VULNERABILITIES_RETENTION_POLICY = '3d';
```

I see two options here:
1. Either we have the retentions match (then let's just reuse existing constants)
2. or if there is a reason to have them different and this is stored somewhere outside of the kibana codebase, let's document it here why exactly we have these numbers

I'd also have `26h` and `3d` documented, because it's not really clear to my why 26h exactly and what it is based on, but that's outside of the scope of the PR

Reply:
In this code, I followed the retention period that was discussed between me and @kfirpeled, How do you think we should proceed @kfirpeled  ?

Reply:
I understand the confusion here, I'll elaborate

The reason for having 5h retention policy for KSPM is because cloudbeat runs a cycle every 4 hours. That was the original retention policy of our transform.

When we introduced CSPM, and because they use the same index. We had to increase the retention policy. However, it degraded KSPM solution in a sence that the retention policy is now increased to 26h. It is not dramtic at all.

So @animehart added here the option to be more precise again with KSPM.
However, if we seek for simplicity I'm ok with not going for that optimization. And use the same retention policy for KSPM and CSPM.

@animehart , @maxcold wdyt?

Reply:
@kfirpeled, I feel like if this optimization is not really 'needed' then I think it might be better to postpone it until we can implement the complete solution for it, because as @opauloh said above, this changes creates inconsistencies between Findings and Dashboard page. That being said, if we are okay with the inconsistency issue caused by this solution, then I'm not opposed with moving forward with these changes

Reply:
I think having the retentions match 
```
export const LATEST_FINDINGS_RETENTION_POLICY = '26h';
export const LATEST_VULNERABILITIES_RETENTION_POLICY = '3d';
```
would do and would be already a good improvement. Yes, for KSPM the dashboard will still be outdated, for `26h` instead of `5h`, but not indefinitely. 
Then we could just reuse these variables and have the connection between these retentions. We will only need to switch to hours vs days for vulnerability one, I hope it won't be too much of a hassle. And also to figure out this `all` retention that still confuses me :) 

Then let's create a follow-up ticket to figure out the retention for KSPM to match the `cloudbeat` cadence.

wdyt @animehart @kfirpeled ?

Reply:
If we decide to go with Optimization, what @maxcold  suggest might be better for us now, considering we would need to add another PR/Ticket even if we chose the other option.

Reply:
> Then let's create a follow-up ticket to figure out the retention for KSPM to match the cloudbeat cadence.

I'm ok with not continuing with a followup ticket here. It doesn't bring any true value from a user's perspective.
I'm happy with your conclusion of simplifying it the way you did

Reply:
> I think having the retentions match
> 
> ```
> export const LATEST_FINDINGS_RETENTION_POLICY = '26h';
> export const LATEST_VULNERABILITIES_RETENTION_POLICY = '3d';
> ```
> 
> would do and would be already a good improvement. Yes, for KSPM the dashboard will still be outdated, for `26h` instead of `5h`, but not indefinitely. Then we could just reuse these variables and have the connection between these retentions. We will only need to switch to hours vs days for vulnerability one, I hope it won't be too much of a hassle. And also to figure out this `all` retention that still confuses me :)

💯 I agree with that as well, and those filtering we can easily apply those to both tables on the findings page.

Reply:
alright, we going to proceed with @maxcold suggestion 👍